### ISSUE TITLE
Cannot Start Concert_Gazebo in docker

### ISSUE THREAD

**Issue by manuelvogel12** [ADVRHumanoids/concert_description#21]
Hello, 
after building and starting the docker image, I tried to start the program with  ```mon launch concert_gazebo concert.launch```
but the following Error occurred:
```
Traceback (most recent call last):
  File "/home/user/concert_ws/ros_src/concert_description/concert_examples/concert_example.py", line 6, in <module>
    urdf_writer = UrdfWriter(speedup=True, floating_base=True)
  File "/home/user/concert_ws/src/modular/src/modular/URDF_writer.py", line 1229, in __init__
    self.base_link = ModuleNode.ModuleNode(data, "base_link")
  File "/home/user/concert_ws/src/modular/src/modular/ModuleNode.py", line 547, in __init__
    super(ModuleNode, self).__init__(dictionary, filename, format=format, template_d=template_dictionary)
  File "/home/user/concert_ws/src/modular/src/modular/ModuleNode.py", line 329, in __init__
    self.interpreter = interpreter_map[format](self, d, filename)
  File "/home/user/concert_ws/src/modular/src/modular/ModuleNode.py", line 298, in __init__
    self.owner.set_size()
  File "/home/user/concert_ws/src/modular/src/modular/ModuleNode.py", line 358, in set_size
    setattr(self, 'size', switcher.get(self.size, "Invalid size"))
AttributeError: can't set attribute
Traceback (most recent call last):
  File "/home/user/concert_ws/ros_src/concert_description/concert_examples/concert_example.py", line 6, in <module>
    urdf_writer = UrdfWriter(speedup=True, floating_base=True)
  File "/home/user/concert_ws/src/modular/src/modular/URDF_writer.py", line 1229, in __init__
    self.base_link = ModuleNode.ModuleNode(data, "base_link")
  File "/home/user/concert_ws/src/modular/src/modular/ModuleNode.py", line 547, in __init__
    super(ModuleNode, self).__init__(dictionary, filename, format=format, template_d=template_dictionary)
  File "/home/user/concert_ws/src/modular/src/modular/ModuleNode.py", line 329, in __init__
    self.interpreter = interpreter_map[format](self, d, filename)
  File "/home/user/concert_ws/src/modular/src/modular/ModuleNode.py", line 298, in __init__
    self.owner.set_size()
  File "/home/user/concert_ws/src/modular/src/modular/ModuleNode.py", line 358, in set_size
    setattr(self, 'size', switcher.get(self.size, "Invalid size"))
AttributeError: can't set attribute
Traceback (most recent call last):
  File "/home/user/concert_ws/ros_src/concert_description/concert_examples/concert_example.py", line 6, in <module>
    urdf_writer = UrdfWriter(speedup=True, floating_base=True)
  File "/home/user/concert_ws/src/modular/src/modular/URDF_writer.py", line 1229, in __init__
    self.base_link = ModuleNode.ModuleNode(data, "base_link")
  File "/home/user/concert_ws/src/modular/src/modular/ModuleNode.py", line 547, in __init__
    super(ModuleNode, self).__init__(dictionary, filename, format=format, template_d=template_dictionary)
  File "/home/user/concert_ws/src/modular/src/modular/ModuleNode.py", line 329, in __init__
    self.interpreter = interpreter_map[format](self, d, filename)
  File "/home/user/concert_ws/src/modular/src/modular/ModuleNode.py", line 298, in __init__
    self.owner.set_size()
  File "/home/user/concert_ws/src/modular/src/modular/ModuleNode.py", line 358, in set_size
    setattr(self, 'size', switcher.get(self.size, "Invalid size"))
AttributeError: can't set attribute
Could not load launch file: /home/user/concert_ws/ros_src/concert_description/concert_gazebo/launch/modular.launch:58: <param> command failed (exit status 1)
```

Sourcing setup.bash also didn't help.
Can you tell me how to proceed? 

Thanks

**Comment by JakobThumm** [comment]
@EdoardoRomiti @alaurenzi Do you face the same issue?

**Comment by EdoardoRomiti** [comment]
Hi @JakobThumm, unfortunately, this is due to an update of the **anytree** library that is used inside `modular`. 
**Pip** now installs version 2.9.0 which apparently has some breaking changes with respect to the previous ones (see similar issues https://github.com/c0fec0de/anytree/issues/222). I was still using the previous version and that's why we didn't catch this issue.
This should now be fixed with commit dfc654c. Re-building / Pulling the docker should do the trick. Let me know if it works now
